scripts/billing/sql/total_costs_by_sku_label.sql

```diff
@@ -0,0 +1,10 @@
+/*
+Query returns total costs by sku and all labels
+*/
+
+SELECT
+  sku.description,
+  TO_JSON_STRING(labels) AS labels,
+  sum(cost) AS cost
+FROM `data-analytics-pocs.public.billing_dashboard_export`
+GROUP BY 1,2
```

Comment:
Prefer explicit column names

Reply:
Updated group by and order to use explicit column names here and across files